src/App.vue

```diff
@@ -95,11 +94,34 @@
 </template>
 
 <style scoped>
+.container {
+  padding: 0;
+}
+.art-header {
+  background-color: #ffffff !important;
+  box-shadow: rgba(0, 0, 0, 0.096) 0px 0px 50px 8px !important;
+}
+.art-footer {
+  background-color: #f9fbfb !important;
+  padding: 23px 0px 23px 0px;
+  border: solid 1px #e4e4e4;
+}
 .logo {
   border-radius: 20%;
   background-color: white;
   margin-left: 20px;
 }
+.art-footerTx1 {
+  color: #126b77 !important;
```

Comment:
Try to avoid `!important`—they can usually be dropped if your styling logic is clear.